src/store/actions/userActions.js

```diff
@@ -0,0 +1,14 @@
+import { 
```

Comment:
Please change your file name to be `user.actions.js`

All redux actions should be named with same convention please